card/mod/standard/set/all/rich_html/content.rb

```diff
@@ -139,46 +138,27 @@ def current_set_card
     Card.fetch(set_name)
   end
 
-  # subheader = with_name_context(card.name) do
-  #   subformat(rcard)._render_title(args)
-  # end
-
-  view :related, cache: :never do |args|
-    related_card, options = related_card_and_options args
+  # the related view nests a related card with a submenu
+  # the subcard is specified as an "item" card using the slot/voo api
```

Comment:
comment above explains api change concept.  it's all about treating the related card as an item (which lets us get rid of a lot of old code)